release: v1.49.0 (#5418)
apollographql/router: .changesets/fix_geal_schema_hash.md

> ### do not hash the entire schema on every query plan cache lookup ([PR #5374](https://github.com/apollographql/router/pull/5374))

Reviewer: ```suggestion
### Skip hashing the entire schema on every query plan cache lookup ([PR #5374](https://github.com/apollographql/router/pull/5374))
```
